src/goto-symex/goto_symex.h

```diff
@@ -590,6 +590,9 @@ class goto_symext
   /// "abc", the symbol will be named "abc_constant_char_array"). Then, the
   /// expression `&sym[0]` is assigned to `char_array` (assuming `sym` denotes
   /// the symbol holding the string data given by `new_char_array`.
+  /// The assignment is preceeded by a assume statements ensuring `length`
+  /// before this call was zero, this is to avoid letting the previous array
```

Comment:
"letting the previous array unconstrained" -> "letting the previous array be unconstrained" or "leaving the previous array unconstrained"